packages/plugin-ext/src/main/node/logs/logs-service.ts

```diff
@@ -0,0 +1,78 @@
+/********************************************************************************
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from  'inversify';
+import { FileSystem } from '@theia/filesystem/lib/common';
+import { join } from 'path';
+import { PluginConf } from '../config/const';
+import URI from '@theia/core/lib/common/uri';
+import { isWindows } from '@theia/core';
+
+export const logServicePath = '/services/logs';
+
+export const LogService = Symbol('LogService');
+export interface LogService {
```

Comment:
To me interface name `LogService` looks confusing. I expect some manipulations with logs, but actually it just provides a directory for storing logs. If you do not plan methods which process logs, I would recommend to use another name.

Reply:
Done. Take a look, please.